public/app/features/plugins/admin/components/PluginDetailsBody.tsx

```diff
@@ -49,6 +50,32 @@ export function PluginDetailsBody({ plugin, queryParams, pageId }: Props): JSX.E
     );
   }
 
+  if (config.featureToggles.externalServiceAccounts && pageId === PluginTabIds.IAM) {
+    return (
+      <Stack direction="column">
+        <Stack direction="row">The {plugin.name} plugin requires the following permissions on your instance:</Stack>
+        <Stack direction="row">
+          <table className="filter-table">
```

Comment:
Would the[ InteractiveTable component ](https://grafana-dev.com/developers/saga/templates/table) work here?

Reply:
Changed as requested! Thanks for the help @Clarity-89